pyproject.toml

```diff
@@ -92,7 +92,17 @@ extend-exclude = '''
 exclude = ["docs/*"]
 
 [tool.ruff.lint]
-select = ["F401", "F541", "I"] # TODO: eventually, expand to other 'D', and other 'F' linting
+# TODO: eventually, expand to other 'D', and other 'F' linting
+select = [
+    "F401",
+    "F541",
+    "I",
+    "ANN001",
+    "ANN201",
+    "ANN202",
+    "ANN205",
+    "ANN206",
```

Comment:
These enforce enforce existence of annotations, which Mypy surprisingly does not do